src/postamble.js

```diff
@@ -68,7 +68,12 @@ if (memoryInitializer) {
     };
     var doBrowserLoad = function() {
       readAsync(memoryInitializer, applyMemoryInitializer, function() {
-        throw 'could not load memory initializer ' + memoryInitializer;
+        var e = new Error('could not load memory initializer ' + memoryInitializer);
```

Comment:
I didn't see any tests on this -- anything I should do to ensure that changing the value thrown from a `string` to an `Error` is fine?

Reply:
We have tests that check failure, but yeah, I believe they don't check the specific error being thrown. Changing this seems fine to me.